projects/packages/my-jetpack/src/class-rest-products.php

```diff
@@ -83,6 +83,27 @@ public function __construct() {
 				),
 			)
 		);
+
+		register_rest_route(
+			'my-jetpack/v1',
+			'site/products/ownership',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => __CLASS__ . '::get_products_by_ownership',
+					'permission_callback' => __CLASS__ . '::permissions_callback',
+				),
+				'schema' => array( $this, 'get_products_schema' ),
+			)
+		);
+
+		$product_arg = array(
```

Comment:
This variable is not used anywhere, and also I don't think `site/products/ownership` has any params to be defined?